meta_capstone/ViewControllers/UIViewController+TileMethods.h

```diff
@@ -15,7 +15,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong) NSMutableArray *tilesArray;
 @property (strong, nonatomic) PFObject *emptyTile;
 @property (strong, nonatomic) NSMutableArray *usableWords;
-@property (nonatomic, strong) NSDictionary *wordCluePairs;
+@property (nonatomic, strong) NSMutableDictionary *wordCluePairs;
```

Comment:
why did you change it to mutable?

Reply:
I add the word clue pairs one by one, so the dictionary changes and needs to be mutable